packages/@uppy/locales/src/ar_SA.js

```diff
@@ -41,7 +41,7 @@ ar_SA.strings = {
   encoding: 'التشفير...',
   enterCorrectUrl: 'خطأ في الرابط: ارجو التأكد من ادخال رابط مباشر للملف',
   enterUrlToImport: 'ادخل الرابط لاستيراد الملفات',
-  exceedsSize: 'الملف اكبر من الحجم المسموح',
+  exceedsSize: 'الملف اكبر من الحجم المسموح %{size}',
```

Comment:
:sweat_smile: It will have to be on the other end

Reply:
I think this is just because the code is rendered in an LTR layout … in an RTL layout, the `%{size}` is at the end of the sentence (so on the left). If we put it on the other side in the code, the size will be the first bytes in the string, but that would be displayed on the right in an RTL layout 🤔 

Our other strings also work this way, so I think this is correct, but we can flip it in a bugfix release if it does turn out to be wrong in the final product.